src/api/config/mod.rs

```diff
@@ -458,3 +475,21 @@ impl Config {
     Ok(seq.tiling)
   }
 }
+
+#[cfg(test)]
+mod tests {
+  use crate::tiling::MAX_TILE_RATE;
+  use crate::{Config, EncoderConfig};
+
+  #[test]
+  #[should_panic(expected = "value: InvalidFrameRate")]
```

Comment:
This is kind of lazy (it'd probably be better to match on the actual type) but I'm not clear on what that would look like.